src/Features/LanguageServer/Protocol/Microsoft.CodeAnalysis.LanguageServer.Protocol.csproj

```diff
@@ -84,13 +88,23 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>LanguageServerResources.resx</DependentUpon>
```

Comment:
I tihnk this can be deleted?  The file doesn't exist in this project.   Must have been a placeholder